src/flashbar/collapsible-flashbar.tsx

```diff
@@ -0,0 +1,351 @@
+                    className={clsx(
+                      animateFlash && styles['flash-with-motion'],
+                      isVisualRefresh && styles['flash-refresh']
+                    )}
+                    key={getItemId(item)}
+                    ref={shouldUseStandardAnimation(item, index) ? transitionRootElement : undefined}
+                    transitionState={shouldUseStandardAnimation(item, index) ? state : undefined}
+                    {...item}
+                  />
+                )}
+              </li>
+            )}
+          </Transition>
+        ))}
+      </TransitionGroup>
+    </ul>
+  );
+
+  return (
+    <div
+      {...baseProps}
+      className={clsx(
+        baseProps.className,
+        styles.flashbar,
+        styles[`breakpoint-${breakpoint}`],
+        styles.stack,
+        isFlashbarStackExpanded && styles.expanded,
+        getVisualContextClassname('flashbar')
+      )}
+      ref={mergedRef}
+    >
+      <>
+        {isFlashbarStackExpanded && renderList()}
+        {isCollapsible && (
+          <div
+            className={clsx(
+              styles.toggle,
+              isVisualRefresh && styles['visual-refresh'],
+              isFlashbarStackExpanded ? styles.expanded : styles.collapsed,
+              transitioning && styles['animation-running']
+            )}
+            onClick={toggleCollapseExpand}
+            ref={toggleElementRef}
+          >
+            <span aria-live="polite" className={styles.status} role="status" id={itemCountElementId}>
+              {toggleButtonText && <h2 className={styles.header}>{toggleButtonText}</h2>}
+              <span className={styles['item-count']}>
+                {counterTypes.map(({ type, labelName, iconName }) => (
+                  <NotificationTypeCount
+                    key={type}
+                    iconName={iconName}
+                    label={i18nStrings ? i18nStrings[labelName] : undefined}
+                    count={countByType[type]}
+                  />
+                ))}
+              </span>
+            </span>
+            <button
+              aria-controls={flashbarElementId}
+              aria-describedby={itemCountElementId}
+              aria-expanded={isFlashbarStackExpanded}
+              aria-label={i18nStrings?.toggleButtonAriaLabel}
+              className={clsx(styles.button, isFlashbarStackExpanded && styles.expanded)}
+              {...isFocusVisible}
+            >
+              <InternalIcon className={styles.icon} size="normal" name="angle-down" />
+            </button>
+          </div>
+        )}
+        {!isFlashbarStackExpanded && renderList()}
+      </>
+    </div>
+  );
+}
+
+const NotificationTypeCount = ({
+  iconName,
+  label,
+  count,
+}: {
+  iconName: IconProps.Name;
+  label?: string;
+  count: number;
+}) => {
+  return (
+    <span className={styles['type-count']}>
+      <span aria-label={label} role="img">
+        <span title={label} aria-hidden="true">
```

Comment:
The `title` attribute is applied to an extra nested element with `aria-hidden` instead of the same element that has `aria-label` in order to prevent NVDA from reading the same text twice.

Reply:
I suspect that NVDA isn't reading it twice now because this element is `aria-hidden`, not because it's a separate element. But either way, it's the result that we want.